workflow-service/src/test/java/com/redhat/parodos/workflow/util/WorkFlowDTOUtilTest.java

```diff
@@ -0,0 +1,80 @@
+package com.redhat.parodos.workflow.util;
+
+import com.redhat.parodos.workflow.execution.dto.WorkFlowRequestDTO;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
should we expand asterisk and list all imported methods?

Reply:
Ideally we can expand them, also hope the google java formatter can auto fix it